packages/blade/src/components/Amount/Amount.tsx

```diff
@@ -0,0 +1,245 @@
+import type { ReactElement } from 'react';
+import type { HeadingProps, TextProps, TitleProps } from '../Typography';
+import {
+  amountFontSizes,
+  currencyAbbreviationsMapping,
+  currencyLocaleMapping,
+  currencyPrefixMapping,
+  affixFontSizes,
+} from './amountTokens';
+import { BaseText } from '~components/Typography/BaseText';
+import type { Feedback } from '~tokens/theme/theme';
+import type { BaseTextProps } from '~components/Typography/BaseText/types';
+import BaseBox from '~components/Box/BaseBox';
+import type { TestID } from '~src/_helpers/types';
+import { metaAttribute, MetaConstants } from '~utils';
+
+type Currency = 'INR' | 'MYR';
+
+type AmountProps = {
+  /**
+   * The value to be rendered within the component.
+   *
+   */
+  value: number;
+  /**
+   * Sets the intent of the amount.
+   *
+   * @default undefined
+   */
+  intent?: Exclude<Feedback, 'neutral'>;
+  /**
+   * Sets the size of the amount.
+   *
+   * @default 'body-medium'
+   */
+  size?:
+    | `title-${NonNullable<Exclude<TitleProps['size'], 'large'>>}`
+    | `heading-${NonNullable<Exclude<HeadingProps<{ variant: 'regular' }>['size'], 'medium'>>}`
+    | `heading-${NonNullable<
+        Exclude<HeadingProps<{ variant: 'regular' }>['size'], 'medium'>
+      >}-${NonNullable<Exclude<TextProps<{ variant: 'body' }>['weight'], 'regular'>>}`
+    | `body-${NonNullable<Exclude<TextProps<{ variant: 'body' }>['size'], 'xsmall'>>}`
+    | `body-${NonNullable<Exclude<TextProps<{ variant: 'body' }>['size'], 'xsmall'>>}-${NonNullable<
+        Exclude<TextProps<{ variant: 'body' }>['weight'], 'regular'>
+      >}`;
+  /**
+   * Indicates what the suffix of amount should be
+   *
+   * @default 'decimals'
+   */
+  suffix?: 'decimals' | 'none' | 'humanize';
+  /**
+   * in amount by making the prefix symbol and decimal digits s
```

Comment:
This doesn't seem to be a legible sentence. Looks like something messed up in merges.